sql/src/test/java/org/opensearch/sql/sql/antlr/SQLSyntaxParserTest.java

```diff
@@ -193,6 +193,12 @@ public void can_parse_now_like_functions(String name, Boolean hasFsp, Boolean ha
 
   @Test
   public void can_parse_multi_match_relevance_function() {
+    assertNotNull(parser.parse(
+        "SELECT id FROM test WHERE multi_match(\"fields\"=\"field\", \"query\"=\"query\")"));
```

Comment:
I thought fields had to be a list

Reply:
I can't seem to get `fields` to be a list with the legacy engine syntax. Syntax like `[field1, field2]` fails the parser with:
```
{
  "error": {
    "reason": "Invalid SQL query",
    "details": "All items between Brackets should be identifiers , got:COMMA",
    "type": "ParserException"
  },
  "status": 400
}
```
The legacy engine does allow space separated fields, but it just concats them into one field:
`['field1' 'field2']`
yields
`"fields": ["field1 field2^1.0"]`

My only thought would be fields is supposed to be a wildcard query with this syntax?

Reply:
Maybe it could be a comma-separated list, like `"fields"="Tags,Title"`. For sure it can't be `"fields"=["Tags", "Title"]`, because it breaks compatibility with legacy.
Actually, it could support both formats, but first one is mandatory.

Reply:
@Yury-Fridlyand
Thanks that worked!